addons/mass_mailing/views/snippets_themes.xml

```diff
@@ -77,7 +77,7 @@
                     <t t-snippet="mass_mailing.s_references" t-thumbnail="/mass_mailing/static/src/img/snippets_thumbs/s_references.svg"/>
                     <t t-snippet="mass_mailing.s_coupon_code" t-thumbnail="/mass_mailing/static/src/img/blocks/block_discount2.png"/>
                     <t t-snippet="mass_mailing.s_mail_block_discount1" t-thumbnail="/mass_mailing/static/src/img/blocks/block_discount1.png"/>
-                    <t t-snippet="mass_mailing.s_mail_block_event" t-thumbnail="/mass_mailing/static/src/img/blocks/block_event.png"/>
```

Comment:
> Is this some sort of untested WIP and you are just asking about the migration part before testing and finishing the PR, or I am missing something obvious? Since the PR is not in draft, not sure.. I considered it done, my bad if this is not the case.
> 
> * It "crashes" (infinite loading) when changing column number: https://watch.screencastify.com/v/FrbMn0DZ2jCK0IgjptFL
> * I can't drag & drop anything anymore if I drag & drop the new event snippet: https://watch.screencastify.com/v/EloOB4HhxARIAeLtm8eH
> * It "crashes" (infinite loading) when clicking on the 404 image: https://watch.screencastify.com/v/as0LkkpjjIEVhtrF5fQO
> 
> About the new images, why is it gray on the PR and can't be opened on my IDE? ![image](https://user-images.githubusercontent.com/30048408/167152030-44d7fafa-3000-4e3f-ab30-731f3de26094.png) ![image](https://user-images.githubusercontent.com/30048408/167152052-7116ce5a-a814-4f4d-ab7d-8f0fb75d730a.png)

Hi Romain, Thanks for reviewing this. Although that wasn't what was asked ;) 
It was just a question about the potential need for an upgrade script.
Quite a lot is moving around with mass_mailing right now and since the rebase I see there are a couple of issues indeed. 

As for the lingering block_event.png, that's my mistake for leaving it there.

Concerning the `data-snippet` attribute, after discussion with AGE (quite a while back now) it had been said we needed it on every new snippet. This might have changed since, or I might have misunderstood. I'll check with him.
I'll look into placing the snippet class first 👍

Thanks again for your input.